_schemaV2.graphql

```diff
@@ -2287,6 +2287,24 @@ enum ArtistSorts {
   TRENDING_DESC
 }
 
+# An artists rail section in the home view
+type ArtistsRailSection implements GenericHomeViewSection & Node {
+  # The component that is prescribed for this section
+  component: HomeViewComponent
+
+  # A globally unique ID.
+  id: ID!
+
+  # A type-specific ID likely used as a database ID.
+  internalID: ID!
+
+  # A stable identifier for this section
+  key: String!
```

Comment:
My first thought was "why do we need a key when we have an ID?" and then I realized that this was more like a slug. Maybe this should be `slug` instead?